StitchCore/StitchCore/Source/Auth/Providers/Anonymous/AnonymousAuthProvider.swift

```diff
@@ -3,7 +3,7 @@ import ExtendedJson
 /// AnonymousAuthProvider provides a way to authenticate anonymously.
 public struct AnonymousAuthProvider: AuthProvider {
     /// The authentication type for anonymous login.
-    public let type: String = "anon-user"
+    public let type: String = AuthProviderTypes.anonymous.rawValue
```

Comment:
[minor] All of these can probably be `AuthProviderTypes` now. I'd like to clean this `protocol` up a bit in general, but there's no reason we should be declaring the rawValue for each of these types.